.github/workflows/prhtmlgenerator.yml

```
@@ -0,0 +1,57 @@
+name: 'PR Testing'
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test-pr:
+    name: Generate HTML
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out self
+        uses: actions/checkout@v2.3.2
+      - name: Check out rss-bridge
+        run: |
+          PR=${{github.event.number}};
+          mv .github/prtester-requirements.txt ./requirements.txt;
+          mv .github/prtester.py .;
+          git clone https://github.com/RSS-Bridge/rss-bridge.git --depth 1;
+          cd rss-bridge;
+          wget https://patch-diff.githubusercontent.com/raw/RSS-Bridge/rss-bridge/pull/$PR.patch;
+          cat $PR.patch | patch -p1;
+          cat $PR.patch | grep " bridges/.*\.php" | sed "s= bridges/\(.*\)Bridge.php.*=\1=g" | sort | uniq > whitelist.txt
+      - name: Start Docker - Current
+        run: |
+          docker run -d -v $GITHUB_WORKSPACE/rss-bridge/whitelist.txt:/app/whitelist.txt -p 3000:80 ghcr.io/rss-bridge/rss-bridge:latest
+      - name: Start Docker - PR
+        run: |
+          docker build -t prbuild $GITHUB_WORKSPACE/rss-bridge/;
+          docker run -d -v $GITHUB_WORKSPACE/rss-bridge/whitelist.txt:/app/whitelist.txt -p 3001:80 prbuild
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+          cache: 'pip'
+      - name: Install requirements
+        run: |
+          cd $GITHUB_WORKSPACE
+          pip install -r requirements.txt
+      - name: Run bridge tests
+        run: |
+          mkdir results;
+          python prtester.py
+      - name: Find result files
+        uses: tj-actions/glob@v7.10
+        id: artifacts
+        with:
+          files: |
+            ./results/*.html
+      - name: Upload results to PR
+        uses: gavv/pull-request-artifacts@v1.0.0
```

I'm not sure how actions work exactly, but I forked the one used in this PR, changed a [line](https://github.com/yamanq/pull-request-artifacts/commit/516aeb03b7bbf393b67069ff1ace29f4f277ccef), and tagged it. Would this work then?

```suggestion
        uses: yamanq/pull-request-artifacts@v1.1.0
```

---

I think you changed it in  the wrong line. Try to build the string in [line 156](https://github.com/yamanq/pull-request-artifacts/blob/516aeb03b7bbf393b67069ff1ace29f4f277ccef/src/main.ts#L156)

---

Thanks, fixed and tested this time. See [test](https://github.com/yamanq/dmxprbldr/pull/1/) and [required changes](https://github.com/yamanq/dmxprbldr/pull/1/files).

---

Very cool, thanks! I will adap the action. I currently have another issue (as you can see from the actions) that there seems to be a permission issue somewhere thats not happening in my own branch.